### ISSUE TITLE
No Status Information Returned

### ISSUE THREAD

**Issue by trailsurfer604** [gogades/hass-infinitive#9]
Control of thermostat works, but no status information is returned.

Here is what I get via dev tools:
```
hvac_modes: off, heat, cool, heat_cool
min_temp: 7
max_temp: 35
fan_modes: auto, low, medium, high
preset_modes: home, Hold
current_temperature: null
temperature: 19
fan_mode: null
preset_mode: home
current_humidity: null
target_humidity: null
blower_rpm: null
stage: null
override_duration: null
airflow_cfm: null
outdoor_temp: null
aux_heat: null
heatpump_coil_temp: null
heatpump_outside_temp: null
heatpump_stage: null
friendly_name: Infinity Thermostat
supported_features: 25
```
I am running Home Assistant 2021.10.7 with latest 0.0.8 infinitive HA integration.

Web GUI does provide this information so it's obviously being read.  Just for some reason not being passed onto the integration.

![image](https://user-images.githubusercontent.com/75198746/140552679-1377f1de-72f6-44f5-b620-af438c42d0d2.png)

**Comment by trailsurfer604** [comment]
Upon further digging, I realized that what is happening is that one of 3 types of config status is being returned.

The following is fro pyinfinitive.py, and only configstatus is return, heandlerstatus and heatpumpstatus are returned empty

```
 def get_status(self):
        """Return current status of an infinitive device."""
        configstatus = self._get_configstatus(self.config_url, '')
        handlerstatus = self._get_configstatus(self.airhandler_url, '')
        heatpumpstatus = self._get_configstatus(self.heatpump_url, 'heatpump_')
        
        return {**configstatus, **heatpumpstatus, **handlerstatus}
```

GUI continues to work fine ...

**Comment by mww012** [comment]
Hey, I commented on your thread in the HA forums before I saw this issue. You're welcome to continue the conversation here if you'd prefer. 

First thing we want to do is make sure infinitive itself is actually getting the data from the unit.

You can hit the api endpoints from a browser and get the json response from infinitive. That will at least let us know if we have an infinitive issue or there's something going on with pyinfinitive.

**Comment by trailsurfer604** [comment]
Hi,

Thank you for reaching out!

I've narrowed the issue down to the actual infinitive instance running on
r-pi.  Although I do have a heat pump and an air handler unit, going to
those JSON URLs (/api/airhandler and /api/heatpump) generates a 404 on the
server side.  I checked server logs and yes, that's what it does.

Unfortunately, there seems to be nothing to configure on that side, and no
way to even play with its operation other than learning GO and
recompiling.  I would love to reach out to the author, but I think he's
abandoned this project.

Thank You,

----------
Seva
